build.gradle.kts

```
@@ -3,160 +3,51 @@
 import kotlinx.benchmark.gradle.KotlinJvmBenchmarkTarget
 
 plugins {
-    kotlin("multiplatform")
-    kotlin("plugin.serialization")
+    buildsrc.conventions.lang.`kotlin-multiplatform-js`
+    buildsrc.conventions.lang.`kotlin-multiplatform-jvm`
+    buildsrc.conventions.lang.`kotlin-multiplatform-native`
+    buildsrc.conventions.publishing
     id("org.jetbrains.kotlinx.benchmark")
     id("org.jetbrains.dokka")
-    `maven-publish`
-    signing
 }
 
 group = "io.github.quillraven.fleks"
 version = "2.3"
```

Do you want to update the version?

```suggestion
version = "2.4-SNAPSHOT"
```